### PR TITLE
[FEAT] Easter Event

### PR DESCRIPTION
**Pull request by adamhannigan**
# Description

Easter event launching on 31st! 🚀 Hopper is searching for his lost rabbits, if you can find them and then feed them you will get a daily reward.

This will go live for Beta testers earlier.

**TODO**

Rabbit placement positions - these are being determined by community. For now they are just above Pete.

## How to test?

1. Run the API
2. Search plaza for 6 rabbits and click on them
3. Deliver carrots for rewards

Screenshot 2024-03-27 at 4 38 47 pm

### CODE REVIEW

File: src/features/world/ui/npcs/Hopper.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  RABIBTS_FOUND += 1;
+}
+
+export const Hopper: React.FC<Props> = ({ onClose }) => {
+  const { gameService } = useContext(Context);
+  const [gameState] = useActor(gameService);
+
+  const { t } = useAppTranslation();
+
+  if (RABIBTS_FOUND < RABBITS) {
```
Comment (brian-man) [inline src/features/world/ui/npcs/Hopper.tsx:32]: `RABIBTS_FOUND` => `RABBITS_FOUND`

File: src/features/world/ui/npcs/Hopper.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    return (
+      <CloseButtonPanel bumpkinParts={NPC_WEARABLES.hopper}>
```
Comment (spencerdezartsmith) [inline src/features/world/ui/npcs/Hopper.tsx:34]: This modal doesn't allow you to close it once Hopper asks you to help find the bunnies. You can click out of the modal but I found myself clicking inside the modal a couple of times wondering why it wasn't closing. 🤷🏼

File: src/lib/i18n/dictionaries/englishDictionary.ts [hunk 3]
```diff
@@ -4150,6 +4152,9 @@ const somethingWentWrong: Record<SomethingWentWrong, string> = {
 };
 
 const specialEvent: Record<SpecialEvent, string> = {
+  "special.event.easterIntro":
```
Comment (spencerdezartsmith) [inline src/lib/i18n/dictionaries/englishDictionary.ts:4155]: Should Hopper mention how many are missing? Once you find them all there is no indication that you have found them all except for going all the way back to Hopper to check. Might be worth either showing how many or indicating when you find the last one with a little 🎉

**Review by spencerdezartsmith — DISMISSED** [review]
LGTM. Non blocking comments only. 🐰

**Pull request merged**